src/devices/innr.ts

```diff
@@ -684,6 +685,30 @@ const definitions: Definition[] = [
             }
         },
     },
+    {
```

Comment:
Can you try with the following instead:

```
{
        zigbeeModel: ['SP 240'],
        model: 'SP 240',
        vendor: 'Innr',
        description: 'Smart plug',
        extend: [
            onOff(),
            electricityMeter(),
        ],
    },
```

Reply:
it works but...
looking at the "exposes" screen of the home assistant/zigbee2mqtt it shows adjusters on the right side of the screen as if you could change voltage.  (see attachment)
![innrsp240exposescreen](https://github.com/Koenkk/zigbee-herdsman-converters/assets/31622045/746e1010-d142-4fe0-8ea8-7ea6bfc11750)

The way I implemented/tested your sample is by converting it into an external converter. (see attachment 2)
![innrsp240](https://github.com/Koenkk/zigbee-herdsman-converters/assets/31622045/fd6e20df-6aeb-43b1-9df8-1a796cbd5d30)

(ps. last time I programmed was in old fashioned assembler, Cobol and later in C. Now learning on how to work with all the modern stuff ;-) )

Reply:
Can you try with:

```js
{
        zigbeeModel: ['SP 240'],
        model: 'SP 240',
        vendor: 'Innr',
        description: 'Smart plug',
        extend: [
            onOff({powerOnBehavior: false}),
            electricityMeter({current: {divisor: 1000}, voltage: {divisor: 1}, power: {divisor: 1}, energy: {divisor: 100}}),
        ],
    },
```

Reply:
Configure goes ok, the powerOnBehavior i tried with and without your suggestion as it already worked. But the function to disable, like in your example, works. So i tested with poweronbehavior enabled and all settings off/on/toggle/restore work.
I noticed that the default Zigbee icon also is replaced by a plug jpeg image.
i think that makes it pretty complete unless you want to test other stuff on it ?